### PR TITLE
Update phel and the new config

### PR DESCRIPTION
**Pull request by Chemaclass**
## 📚 Description

Update phel config 

## 🔖 Changes

- Remove phel config from composer extra key-values
- Add `phel-config.php`

### CODE REVIEW

**Comment by jenshaase** [comment]
Is there an way to get rid of the gacela.php file in future?

**Comment by Chemaclass** [comment]
Regarding this `gacela.php` we can get rid of it IF we use the default config place for Gacela, which is placing the phel-config.php file in a `config/` directory at the root level or the project. In that case, we don't need the `gacela.php` file at all.

I think an easy alternative would be simply moving the `/phel-config.php` file to `config/phel-config.php`. We might need to do some minor adjustments in `phel-lang`, but this would allow us to use the default Gacela config stuff, and the end-user doesn't need to know anything about Gacela.

Worst scenario, we can try this idea and see how do we like it or we should find something else.

Is this idea ok for you, @jenshaase? Or do you have something else in mind?

**Comment by jenshaase** [comment]
I just had a quick look into the Gacela code base. I see two options:

1. Provide a way to customize the way Gacela loads the configuration. Either with a dependency injection container or by given the required classes to the bootstrap method.

2. Assigning the configs that are in `gacela.php` directly to the bootstrap method. Gacela then doesn't have to load the configs from gacela.php.

What do you think?

**Comment by Chemaclass** [comment]
You gave me an idea! 💡  I will try to allow passing the config as a "global service" that you could inject when doing `Gacela::bootstrap()`. Something like:
```php
Gacela::bootstrap($projectRootDir, [
    'config' => [
        'type' => 'php',
        'path' => 'phel-config.php',
        'path_local' => 'phel-config-local.php',
    ],
]);
```
I will work on it now 🚀

File: README.md [hunk 1]
```diff
@@ -29,18 +29,14 @@ You can either use it from your local machine OR using docker.
 ### Phel code
 
 1. Write your phel code in `src/`
-1. Execute your code with `./vendor/bin/phel run src/command/main.phel`
-1. Or with a PHP server with `php -S localhost:8000 -t ./src/web`
+2. Execute your code with `./vendor/bin/phel run src/command/main.phel`
+3. Or with a PHP server with `php -S localhost:8080 src/web/index.php`
```
Comment (jenshaase) [inline README.md]: With the bug fix I just did (in phel-lang), we can change this back to the original value. Then the images and css files are loading fine.

**Pull request merged**